src/shortcut.rs

```diff
@@ -0,0 +1,52 @@
+#[allow(unused_imports)]
```

Comment:
Suppressions like this shouldn't be present. Yo should make things properly conditionally compiled instead. Alternatively you can move import into `{}` (of function) where it is used in a platform-specific way instead.

Reply:
thanks for the guidance